raiden/tests/scenarios/ci/long_path_mediated_transfers.yaml

```diff
@@ -1,365 +1,391 @@
+      enable: true
+      token:
+        # Make sure that enough is deposited to pay for an MR
+        # The cost of an MR is `5 * 10 ** 18`
+        deposit: true
+        balance_per_node: 100_000_000_000_000_000_000
+        min_balance: 5_000_000_000_000_000_000
+
 
 token:
-  decimals: 18
+  address: "0x62083c80353Df771426D209eF578619EE68D5C7A"
+  balance_fund: 10_000_000_000_000_000_000
 
 nodes:
   mode: managed
   count: 15
-  raiden_version: LATEST
+  raiden_version: local
+
   default_options:
     gas-price: fast
     environment-type: development
+    routing-mode: pfs
+    pathfinding-max-fee: 100
+    enable-monitoring: true
+    proportional-fee:
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 0
+    proportional-imbalance-fee:
+      - "0x62083c80353Df771426D209eF578619EE68D5C7A"
+      - 0
+    default-settle-timeout: 40
+    default-reveal-timeout: 20
+
+
+
 
 scenario:
   serial:
     tasks:
       - parallel:
           name: "Open 15 channels in both directions"
           tasks:
-            - open_channel: {from: 0, to: 1, total_deposit: 1000}
-            - open_channel: {from: 1, to: 0, total_deposit: 1000}
-            - open_channel: {from: 1, to: 2, total_deposit: 1000}
-            - open_channel: {from: 2, to: 1, total_deposit: 1000}
-            - open_channel: {from: 2, to: 3, total_deposit: 1000}
-            - open_channel: {from: 3, to: 2, total_deposit: 1000}
-            - open_channel: {from: 3, to: 4, total_deposit: 1000}
-            - open_channel: {from: 4, to: 3, total_deposit: 1000}
-            - open_channel: {from: 4, to: 5, total_deposit: 1000}
-            - open_channel: {from: 5, to: 4, total_deposit: 1000}
-            - open_channel: {from: 5, to: 6, total_deposit: 1000}
-            - open_channel: {from: 6, to: 5, total_deposit: 1000}
-            - open_channel: {from: 6, to: 7, total_deposit: 1000}
-            - open_channel: {from: 7, to: 6, total_deposit: 1000}
-            - open_channel: {from: 7, to: 8, total_deposit: 1000}
-            - open_channel: {from: 8, to: 7, total_deposit: 1000}
-            - open_channel: {from: 8, to: 9, total_deposit: 1000}
-            - open_channel: {from: 9, to: 8, total_deposit: 1000}
-            - open_channel: {from: 9, to: 10, total_deposit: 1000}
-            - open_channel: {from: 10, to: 9, total_deposit: 1000}
-            - open_channel: {from: 10, to: 11, total_deposit: 1000}
-            - open_channel: {from: 11, to: 10, total_deposit: 1000}
-            - open_channel: {from: 11, to: 12, total_deposit: 1000}
-            - open_channel: {from: 12, to: 11, total_deposit: 1000}
-            - open_channel: {from: 12, to: 13, total_deposit: 1000}
-            - open_channel: {from: 13, to: 12, total_deposit: 1000}
-            - open_channel: {from: 13, to: 14, total_deposit: 1000}
-            - open_channel: {from: 14, to: 13, total_deposit: 1000}
+            - open_channel: {from: 0, to: 1, total_deposit: 1_000}
```

Comment:
Can you please update the entire scenario to use higher values like in all other scenarios?

Reply:
Making the change